### PR TITLE
Update meteoalarm.markdown

### PR DESCRIPTION
**Pull request by rolfberkenbosch**
## Proposed change

MeteoAlarm has changed their api and website, so i made a small change in the documentation and intergration to make it work again.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51383
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/meteoalarm.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ ha_platforms:
   - binary_sensor
 ---
 
-The `MeteoAlarm` platform allows one to watch for weather alerts in europe from [MeteoAlarm](https://www.meteoalarm.eu) (EUMETNET). To use this binary sensor, you need the two digits of your country and the province name from  [MeteoAlarm](https://www.meteoalarm.eu). Please note that you need to write the exact details from the website with capitals.
+The `MeteoAlarm` platform allows one to watch for weather alerts in europe from [MeteoAlarm](https://www.meteoalarm.org) (EUMETNET). To use this binary sensor, you need the two digits of your country and the province name from  [MeteoAlarm](https://www.meteoalarm.org). Please note that you need to write the exact details from the website with capitals.
```
Comment (MartinHjelmare) [inline source/_integrations/meteoalarm.markdown]:
```suggestion
The `MeteoAlarm` platform allows one to watch for weather alerts in europe from [MeteoAlarm](https://www.meteoalarm.org) (EUMETNET). To use this binary sensor, you need the name of your country and the province name from  [MeteoAlarm](https://www.meteoalarm.org). Please note that you need to write the exact details from the website with capitals.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by MartinHjelmare** [comment]
This PR is targeting current branch. Will the new country config work without the parent PR?

**Comment by frenck** [comment]
@MartinHjelmare Nope, so this needs to target `next`.

**Review by frenck — APPROVED** [review]
Thanks, @rolfberkenbosch 👍

**Pull request merged**